src/api/kube.ts

```diff
@@ -525,6 +525,18 @@ export class KubeHelper {
     }
   }
 
+  async getConfigMapValue(name: string, namespace: string, key: string): Promise<string | undefined> {
+    const k8sCoreApi = KubeHelper.KUBE_CONFIG.makeApiClient(CoreV1Api)
+    try {
+      const { body } = await k8sCoreApi.readNamespacedConfigMap(name, namespace)
+      if (body.data) {
+        return body.data[key]
+      }
+    } catch {
+      return
```

Comment:
Maybe in the tslint.json we should apply: `"no-empty": [true, "allow-empty-catch", "allow-empty-functions"]` and change code:

```
} catch {}
```
To fix ` Opening brace cannot be followed by empty line`

Reply:
But up to you.